README.md

```diff
@@ -703,20 +704,22 @@ servicesManager.DeleteRepository("generic-repo")
 ```
 
 #### Getting Repository Details
-You can get repository details from Artifactory using its key, and the desired params struct:
+You can get repository details from Artifactory using its key, and the desired params struct.
+The function expect to get the repo key (name) and a pointer to param struct that will be filled up.
```

Comment:
expect --> expects
...pointer to param struct... --> ...pointer to a param struct...